tests/w2w_test.py

```diff
@@ -94,13 +96,11 @@ def test_create_wrf_gridinfo(tmpdir):
     }
     create_wrf_gridinfo(info)
     assert os.listdir(tmpdir) == ['dst_gridinfo.tif']
-    tif = xarray.open_rasterio(info['dst_gridinfo'])
-    assert tif.crs == '+init=epsg:4326'
-    assert tif.transform == pytest.approx(
-        (
+    tif = rioxarray.open_rasterio(info['dst_gridinfo'])
+    assert tif.rio.crs.to_proj4() == '+init=epsg:4326'
+    assert tif.rio.transform() == Affine(
             0.01000213623046875, 0.0, -1.4050254821777344,
             0.0, 0.010000228881835938, 41.480000495910645,
-        )
     )
```

Comment:
As far as I understand the new API the `transform` must be done like this: `d_rxr.rio.transform()`. Many of the attributes are now only accessible via the `.rio` attribute.
I can't say that I really like this new way. Also there is no real migration guide (yet?), so this was fighting a lot of AttributeErrors and reading the docs of the different API to find what was a 1st hierachy attribute before...